### PR TITLE
added more mosfacts

### PR DESCRIPTION
**Pull request by nine-inch-rails**
Added text file that contains more mosfacts

### CODE REVIEW

File: facts.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+MOSFET stands for Metal oxide semiconducter field effect transistor
```
Comment (skinsshark) [inline facts.txt]: Capitalize each word of MOSFET

File: facts.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+MOSFETS operate at higher switching speeds and lower currents compared to BJTs
```
Comment (skinsshark) [inline facts.txt]: change to `MOSFETs` instead of `MOSFETS` everywhere

File: facts.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+MOSFETS require a very small gate current. It is often negligible unless the drain current is relatively small as well
+In 2014, Intel shipped devices with MOSFETS the size of 14nm
+The MOSFET was invented by Dawon Kahng and Martin M. Atalla at Bell labs in 1959
```
Comment (skinsshark) [inline facts.txt]: stylized `Bell Labs`

File: facts.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+MOSFET stands for Metal oxide semiconducter field effect transistor
+MOSFETS operate at higher switching speeds and lower currents compared to BJTs
+MOSFETS require a very small gate current. It is often negligible unless the drain current is relatively small as well
```
Comment (skinsshark) [inline facts.txt]: r u gonna talk bt 242

File: facts.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+In 2014, Intel shipped devices with MOSFETS the size of 14nm
```
Comment (skinsshark) [inline facts.txt]: nm or nym

**Review by skinsshark — CHANGES_REQUESTED** [review]

File: facts.txt [hunk 6]
```diff
@@ -1,8 +1,8 @@
-MOSFET stands for Metal oxide semiconducter field effect transistor
-MOSFETS operate at higher switching speeds and lower currents compared to BJTs
-MOSFETS require a very small gate current. It is often negligible unless the drain current is relatively small as well
-In 2014, Intel shipped devices with MOSFETS the size of 14nm
-The MOSFET was invented by Dawon Kahng and Martin M. Atalla at Bell labs in 1959
+MOSFET stands for Metal Oxide Semiconducter Field Effect Transistor
```
Comment (skinsshark) [inline facts.txt]: typo in Semiconductor

**Review by skinsshark — APPROVED** [review]

File: facts.txt (hunk 6 above)
Comment (nine-inch-rails) [reply facts.txt]: Oops

**Pull request merged**